src/main/resources/schema.sql

```diff
@@ -20,4 +20,14 @@ VALUES
     (2,'Testowy 2','Opis 2'),
     (3,'Testowy 3','Opis 3');
 -- SELECT * FROM USER U
--- JOIN ROLE R ON U.ROLE_ID = R.ID
\ No newline at end of file
+-- JOIN ROLE R ON U.ROLE_ID = R.ID
+
+INSERT INTO NOTIFICATION_TEMPLATE(ID, NAME, SUBJECT, MESSAGE)
```

Comment:
**info** Storing the templates in DB is totally fine. IMO it's worth mentioning that it could be also stored in:
- some XXX.properties file which would be simpler solution and Spring would be able to support internationalization (i18n) for different languages
- or even in Java class (enum) (why not?) which would be the simplest solution according to KISS (I know it might sound controversial but what's wrong with storing Strings in Java class?).

I know you might want to be able to change templates in the runtime (through SQL update) without a need to redeploy your service but frankly speaking: Is the template going to change so frequently that you need such powerful ability?

Reply:
Thanks. So should I change it or can I leave it the way it is?

Reply:
No, no, you don't have to change, I only wanted to present all approaches and discuss pros/cons, IMO simplicity (KISS & YAGNI principles) is important and I wanted to signal that your business requirements could have been achieved by using also some simpler solutions.